peak/float.py

```diff
@@ -52,373 +55,222 @@ def to_rm(rm: BitVector[3]) -> RoundingMode:
 
     return SimpleNamespace(**locals())
 
+
+def _format_name(name: str) -> str:
```

Comment:
To not break any code that might be using this already  I make sure the names haven't change